geode-core/src/main/java/org/apache/geode/internal/cache/ClusterConfigurationLoader.java

```diff
@@ -94,26 +92,16 @@ public void deployJarsReceivedFromClusterConfiguration(ConfigurationResponse res
             .flatMap(Set::stream)
             .collect(Collectors.toList());
 
-    if (jarFileNames != null && !jarFileNames.isEmpty()) {
+    if (!jarFileNames.isEmpty()) {
       logger.info("Got response with jars: {}", jarFileNames.stream().collect(joining(",")));
```

Comment:
This line can be replaced with `logger.info("Got response with jars: {}", String.join(",", jarFileNames));`